components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -57,6 +57,8 @@ @implementation MDCCollectionViewCell {
   UIImageView *_editingSelectorImageView;
 }
 
+@synthesize inkView = _inkView;
```

Comment:
If we override both the getter and setter of a property, you have to synthesize the ivar yourself. From Apple Docs:

> Note: The compiler will automatically synthesize an instance variable in all situations where it’s also 
> synthesizing at least one accessor method. If you implement both a getter and a setter for a readwrite 
> property, or a getter for a readonly property, the compiler will assume that you are taking control over the 
> property implementation and won’t synthesize an instance variable automatically.
> If you still need an instance variable, you’ll need to request that one be synthesized:
> `@synthesize property = _property;`